app/uk/gov/hmrc/nisp/views/main.scala.html

```diff
@@ -209,7 +209,7 @@ <h1 class=@headerClass> @Html(title) </h1>
     if (mainTemplateParams.h1Class.isDefined){
         mainTemplateParams.h1Class.get
     } else {
-        "heading-large top-title-heading heading-bottom-margin"
+        mainTemplateParams.h1Class.getOrElse("heading-large top-title-heading heading-bottom-margin")
```

Comment:
You can remove whole if and put only
 
mainTemplateParams.h1Class.getOrElse("heading-large top-title-heading heading-bottom-margin")

Reply:
Will look like:
 @headerClass = @{
    mainTemplateParams.h1Class.getOrElse("heading-large top-title-heading heading-bottom-margin")
}

Reply:
cheers mate, i'll make the changes now to the templates